[1,"\u001B[31m\u001B[91mERROR: The specified directory ('.') is not a Quarto project\u001B[31m\n"]
[1,"\u001B[39m"]
[1,"\u001B[1m\u001B[33mError\u001B[39m in `quarto::quarto_render()`:\u001B[22m\n"]
[1,"\u001B[1m\u001B[22m\u001B[31m✖\u001B[39m Error running quarto cli.\n"]
[1,"\u001B[1mCaused by error:\u001B[22m\n"]
[1,"\u001B[33m!\u001B[39m System command 'quarto' failed\n"]
[1,"Backtrace:\n"]
[1,"\u001B[90m     \u001B[39m▆\n"]
[1,"\u001B[90m  1. \u001B[39m└─\u001B[1mglobal\u001B[22m sourceWithProgress(...)\n"]
[1,"\u001B[90m  2. \u001B[39m  └─\u001B[1mbase\u001B[22m::eval(statements[[idx]], envir = sourceEnv)\n"]
[1,"\u001B[90m  3. \u001B[39m    └─base::eval(statements[[idx]], envir = sourceEnv)\n"]
[1,"\u001B[90m  4. \u001B[39m      └─\u001B[1m\u001B[94mquarto::quarto_render(\u001B[39m\u001B[22m\u001B[1m\u001B[94m)\u001B[39m\u001B[22m\n"]
[1,"\u001B[90m  5. \u001B[39m        └─quarto:::quarto_run(args, echo = TRUE, quarto_bin = quarto_bin)\n"]
[1,"\u001B[90m  6. \u001B[39m          └─\u001B[1mbase\u001B[22m::tryCatch(...)\n"]
[1,"\u001B[90m  7. \u001B[39m            └─base (local) tryCatchList(expr, classes, parentenv, handlers)\n"]
[1,"\u001B[90m  8. \u001B[39m              └─base (local) tryCatchOne(expr, names, parentenv, handlers[[1L]])\n"]
[1,"\u001B[90m  9. \u001B[39m                └─value[[3L]](cond)\n"]
[1,"\u001B[90m 10. \u001B[39m                  └─\u001B[1mcli\u001B[22m::cli_abort(msg, call = .call, parent = e)\n"]
[1,"\u001B[90m 11. \u001B[39m                    └─\u001B[1mrlang\u001B[22m::abort(...)\n"]
[2,"Execution halted\n"]
